src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -0,0 +1,786 @@
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard13 { get; }
+                = new ReferenceAssemblies("netstandard1.3")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard14 { get; }
+                = new ReferenceAssemblies("netstandard1.4")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard15 { get; }
+                = new ReferenceAssemblies("netstandard1.5")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard16 { get; }
+                = new ReferenceAssemblies("netstandard1.6")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard20 { get; }
+                = new ReferenceAssemblies(
+                    "netstandard2.0",
+                    new PackageIdentity(
+                        "NETStandard.Library",
+                        NuGetVersion.Parse("2.0.3")),
+                    @"build\netstandard2.0\ref")
+                .AddAssemblies(ImmutableArray.Create("netstandard"));
+
+            ////public static ReferenceAssemblies NetStandard21 { get; }
+            ////    = new ReferenceAssemblies("netstandard2.1")
+            ////    .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("2.0.3"))));
+        }
+
+#if false
+        public static class Runtime
+        {
+            public static ReferenceAssemblies Current
+            {
+                get
+                {
+                    var metadataReferences =
+                        new[]
+                        {
+                            typeof(object).GetTypeInfo().Assembly,
+                            typeof(System.Diagnostics.Debug).GetTypeInfo().Assembly,
+                            typeof(Enumerable).GetTypeInfo().Assembly,
+                            typeof(Microsoft.VisualBasic.Strings).GetTypeInfo().Assembly,
+                        }
+                        .Select(assembly => assembly.Location)
+                        .Distinct()
+                        .Select(location => MetadataReferences.CreateReferenceFromFile(location))
+                        .ToImmutableArray();
+
+                    return new ReferenceAssemblies(
+                        metadataReferences,
+                        ImmutableDictionary.Create<string, ImmutableArray<MetadataReference>>());
+                }
+            }
+        }
+#endif
```

Comment:
I don't think RID should effect analyzers (well it might but I think that can be out of scope for now)

---
In reply to: [341207005](https://github.com/dotnet/roslyn-sdk/pull/329#discussion_r341207005) [](ancestors = 341207005)

Reply:
I removed this block of excluded code